### DISCUSSION TITLE
Error with repository snapshot

### DISCUSSION THREAD

**Discussion by teutonicod** [Open-Systems-Pharmacology/Forum#901] [Questions & Problems] [answered]
Dear all,

I tried to import a snapshot available on the repository but got an error message

https://github.com/Open-Systems-Pharmacology/Erythromycin-Carbamazepine-DDI

What I did was just to download the snapshot and try to load it in PK-Sim. This is what I got

![image](https://user-images.githubusercontent.com/11489621/119136504-0507c280-ba40-11eb-9758-7d2ad2eac0c6.png)

Is this related to a change in PK-Sim version, or a problem with the snapshot?

Best,

Donato

**Comment by Yuri05** [comment, marked as answer]
I think you downloaded not the json but the [html](https://github.com/Open-Systems-Pharmacology/Erythromycin-Carbamazepine-DDI/blob/main/Erythromycin-Carbamazepine-DDI.json) 😉 Click on the "Raw" button to download the [json](https://raw.githubusercontent.com/Open-Systems-Pharmacology/Erythromycin-Carbamazepine-DDI/main/Erythromycin-Carbamazepine-DDI.json) file

**Reply by teutonicod** [reply]
Hi @Yuri05 , 
thanks for your reply. When downloading the file I just right-clik on the file and chose "Save as", and it was downloaded as jason file. But I suppose this was not the right was. I am not sure how to have the "Raw" version, since in that case the file is opened on the browser. What I finally did was to download the zip file and it worked!

Thanks a lot again!

Donato

**Reply by Yuri05** [reply]
Just click on the "Raw" button - which will open the file in your browser; then use SaveAs to save the file.

**Reply by teutonicod** [reply]
Got it! Thanks!
